Tests/Common/PasswordManipulationTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Framework\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Zest\Common\PasswordManipulation;
+
+class PasswordManipulationTest extends TestCase
+{
+    public function testGeneratePassword()
+    {
+        $pass = new PasswordManipulation();
+        $this->assertNotEmpty($pass->generatePassword(11));
+    }
+
+    public function testCountTes()
+    {
+        $pass = new PasswordManipulation();
+        $this->assertSame(6, $pass->len(123456));
+    }
+
+    public function testIsValid()
+    {
+        $pass = new PasswordManipulation();
+        $this->assertTrue($pass->isValid($pass->generatePassword(15)));
+        $this->assertFalse($pass->isValid('1234dasd'));
+    }
+
+    public function testSetLength()
+    {
+        $pass = new PasswordManipulation();
+        $pass->setLength(21);
+        $this->assertNotEmpty($this->getLength());
+    }
+
+    public function testGetLength()
+    {
+        $pass = new PasswordManipulation();
+        $this->assertNotEmpty($this->getLength());
```

Comment:
It should be `$pass->getLength`.

The Travis CI build is failed. They're as follows:

```
1) Framework\Tests\PasswordManipulationTest::testSetLength

Error: Call to undefined method Framework\Tests\PasswordManipulationTest::getLength()

/home/travis/build/zestframework/Zest_Framework/Tests/Common/PasswordManipulationTest.php:33

2) Framework\Tests\PasswordManipulationTest::testGetLength

Error: Call to undefined method Framework\Tests\PasswordManipulationTest::getLength()
```